### PR TITLE
UXUI-539, Adding a SAGA Theme

### PR DESCRIPTION
**Pull request by elliottcrush**
#### What does this PR do? (please provide any background)
We want a boiler plate SAGA theme base, this showcases the work myself and @stuartgiles did on demo day. This is by no means a finished brand but certainly a good place to start. I will create a github issue to complete further work on this.

#### What tests does this PR have?
None

#### How can this be tested?
Checkout the branch in pattern library and feel free to add notes to the github issue on what needs adding. The benefit of the PL is that we can instantly see what needs adding in the brand.

---

- [x] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

#### Reviewers

**Review 1**
- [x] :+1:

**Review 2** \*
- [ ] :+1:

**Review 3** _(optional)_
- [ ] :+1:

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru

### CODE REVIEW

File: brands/saga/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@holidayextras/saga-brand",
+  "version": "1.0.2",
```
Comment (jodiedoubleday) [inline brands/saga/package.json]: has there already been a `1.0.0` and `1.0.1` ?

Comment (elliottcrush) [inline brands/saga/package.json]: Unfortunately yes from the initial publish to setup on npm

**Comment by elliottcrush** [comment]
Related Issue @nassimdonald : https://github.com/holidayextras/brands/issues/22

**Comment by nassimdonald** [comment]
All working for me. A few things missing:
* `blockquote footer` seems to have a background-color inherited from base `footer` styles.
* The footers are not full-width - e.g. in atoms > progress-bar.
* `hr` style variations don't visibly vary from one another.
* Jumbotron covers don't appear to be styled in the saga style.
* In list-group items with active state, child links aren't visible.
* Missing owl carousel styling.
* Content is lost under navbar - visible in pages > payment.
* Navbar links appear very light grey and white (invisible) on hover/focus.

**Pull request merged**